src/sentry/receivers/sentry_apps.py

```diff
@@ -59,7 +60,6 @@ def send_issue_resolved_webhook(organization_id, project, group, user, resolutio
 def send_issue_ignored_webhook(project, user, group_list, **kwargs):
     for issue in group_list:
         send_workflow_webhooks(project.organization, issue, user, "issue.ignored")
-        send_workflow_webhooks(project.organization, issue, user, "issue.archived")
```

Comment:
we deprecated `issue.ignored` in favor of `issue.archived` but kept sending both for backwards compatibility. The way this worked was that the old installations were subscribe to `ignored` only and the new ones to `archived` only so no one would get 2 webhooks for one event. Instead of sending 2 webhooks I'm just renaming it to what the installation expects in line 142.